setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # setup script
-# Copyright (C) 2019, 2020  Nguyễn Gia Phong
+# Copyright (C) 2019, 2020  Nguyễn Gia Phong, Francesco Caliumi
```

Comment:
We have copyright notice for each author on a separate line. Yours would look like this:

```python
# Copyright (C) 2020  Francesco Caliumi
```